Improve Webservices section of the documentation (#1673)
PrestaShop/docs: webservice/getting-started.md

To edit an existing resource: **GET** the full XML file for the resource you want to change (example `/api/addresses/1`), edit its content as needed, then send a **PUT HTTP request** with the whole XML file as a body content to the same URL again.

{{% notice warning %}}

> Make sure your content does not includes any formating (`\n`, `\t`, ...) in the `xml` tags, otherwise it can break your resource.

Reviewer: ```suggestion
Make sure your content does not include any formatting (`\n`, `\t`, ...) in the `xml` tags, otherwise, it can break your resource.
```

> Good input: 

Reviewer: ```suggestion
Correct input: 
```

```
<id>{id}</id>
```

> Wrong input: 

Reviewer: ```suggestion
Incorrect input: 
```
